challenge-03/README.md

```diff
@@ -150,8 +178,28 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function() {
+	if (pessoa.sexo === 'Feminino'){
+		novoSexo = 'a';
+	} else {
+		novoSexo = 'o';
+	} if (pessoa.idade === 1) {
+		novaIdade = 'ano';
+	} else {
+	    novaIdade = 'anos';
+	} if(pessoa.caminhoQuantosMetros === 1) {
+		novoMetro = 'metro';
+	} else {
```

Comment:
-1
Uma dica nessa parte: declare as variáveis no início da função, com valores padrão. Dessa forma, você não precisará de nenhum `else`, somente do `if` para verificar se o valor deve mudar ou não :)